EpiAware/src/latent-processes.jl

```diff
@@ -5,18 +5,14 @@ function default_rw_priors()
     )
 end
 
-@model function random_walk(
-    n,
-    ϵ_t = missing,
-    ::Type{T} = Float64;
-    latent_process_priors = default_rw_priors(),
-) where {T<:AbstractFloat}
-    rw = Vector{T}(undef, n)
+@model function random_walk(n; latent_process_priors = default_rw_priors())
     ϵ_t ~ MvNormal(ones(n))
     σ²_RW ~ latent_process_priors.var_RW_dist
     init ~ latent_process_priors.init_rw_value_dist
     σ_RW = sqrt(σ²_RW)
+    rw = Vector{eltype(ϵ_t)}(undef, n)
 
+    # rw = σ_RW * cumsum(ϵ_t)
```

Comment:
either remove or justify why we need commented out code

Reply:
That was a mistake leave. Pushed the removed.